db/stats.py

```diff
@@ -191,40 +177,20 @@ def format_statistics_for_highcharts(data):
 
 
 def load_statistics_data(limit=None):
-    # Postgres doesn't let you create a json dictionary using values
-    # from one column as keys and another column as values. Instead we
-    # create an array of {"name": name, "value": value} objects and change
-    # it in python
     args = {}
-    qtext = """
-            SELECT collected
-                 , json_agg(row_to_json(
-                    (SELECT r FROM (SELECT name, value) r) )) AS stats
-              FROM statistics
-             WHERE date_part('hour', timezone('UTC'::text, collected)) = 0 
-                OR collected = (SELECT collected
-                                  FROM statistics
-                              ORDER BY collected DESC
-                                 LIMIT 1)
-          GROUP BY collected
-          ORDER BY collected DESC
+    qtext = """SELECT collected, stats
+                 FROM statistics
+             ORDER BY collected DESC
```

Comment:
todo: restore adding the most up-to-date cached value to the end of the stats here when drawing the graph